modules/standard/Map.chpl

```diff
@@ -380,16 +378,22 @@ module Map {
                this map.
     */
     iter items() const ref {
-      for key in myKeys {
-        yield (key, vals[key]);
+      for slot in table.allSlots() {
+        if table.isSlotFull(slot) {
+          ref tabEntry = table.table[slot];
+          yield (tabEntry.key, tabEntry.val);
+        }
       }
     }
 
     pragma "no doc"
     iter items() const ref where isNonNilableClass(valType) {
```

Comment:
I think that with `chpl__hashtable` as the implementation, it should be possible to remove this overload.